selectrum.el

```diff
@@ -182,6 +182,19 @@ It gets the same arguments as `selectrum-read' got, prepended
 with the string the user inserted."
   :type 'hook)
 
+(defcustom selectrum-count-style 'current/matches
+  "The style to use for displaying count information before the prompt.
+
+Possible values are:
+
+- 'matches: Show total number of matches.
+- 'current/matches: Show the index of current match and total number of matches.
```

Comment:
Use `\\='` instead of `'` to make the quote show up correctly in `*Help*`.

Reply:
Done. Actually I've been searching for this for a while. Where did you find this information?

Reply:
The docstring of `substitute-command-keys`, which I found by complete coincidence one day. It's not discoverable.